vizro-core/docs/pages/user-guides/pages.md

```diff
@@ -21,11 +21,11 @@ A [`Page`][vizro.models.Page] is split up into four main containers:
 
 To create and add a page to your dashboard, do the following steps:
 
-1. Provide a `title` to your [`Page`][vizro.models.Page]
+1. Set a `title` for your [`Page`][vizro.models.Page]
 2. Configure your `components`, see our guide on the [various options](components.md)
 3. (optional) Configure your `controls` , see our guides on [Filters](filters.md) and [Parameters](parameters.md)
 4. (optional) Configure your `layout` , see our guide on [Layouts](layouts.md)
-5. (optional) Provide a `description` of your `Page` for the app's [meta tags](https://metatags.io/)
+5. (optional) Give a `description` of your `Page` to the app's [meta tags](https://metatags.io/)
```

Comment:
Is this correct now? You are not really giving it to the tags, but you are giving it so that the tags change. Maybe a detail...